app/src/main/java/org/oppia/app/testing/AudioFragmentTestActivity.kt

```diff
@@ -12,6 +16,14 @@ class AudioFragmentTestActivity : InjectableAppCompatActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     activityComponent.inject(this)
-    audioFragmentTestActivityController.handleOnCreate()
+    val internalProfileId = intent.getIntExtra(KEY_AUDIO_FRAGMENT_TEST_PROFILE_ID, 0)
+    audioFragmentTestActivityController.handleOnCreate(internalProfileId)
+  }
+  companion object {
```

Comment:
Nit: please add spacing above per https://developer.android.com/kotlin/style-guide#vertical.

Reply:
Done

Reply:
Checked and its correct. Resolving on behalf of Ben.